### DISCUSSION TITLE
Questão TI21#6

### DISCUSSION THREAD

**Discussion by paulocafee** [paulowh/TI21#12] [Q&A] [answered]
Como você pode implementar herança em CSS?

**Comment by erickgusson** [comment]
Você pode usar usar o inherit
exemplo:

display: inherit;

o display vai ser o mesmo do elemento pai, se você der um display flex no pai ele fica com display flex

**Reply by erickgusson** [reply]
Você também pode usar para os tamanho de fonte
exemplo:

o elemento pai tem a font em 40px
se o filho for posto usando 0.5em a fonte fica com 20px

**Comment by RenonHG** [comment, marked as answer]
Por natureza um elemento dentro de outro, se não tiver atribuído diretamente uma propriedade de cor por exemplo, ele ganhará as propriedades do elemento pai (aquele que ele está dentro), você pode atribuir diretamente propriedades utilizando no css exatamente aquele elemente, #body .p{...} e por aí vai.
